content/user/ui/overview/Basic-quick-start-guide.md

```diff
@@ -0,0 +1,111 @@
+
+To  see all the information, these applications must be licensed. For more information on licensing your application, read the article on [Licensing an Application](https://docs.contrastsecurity.com/user_tsguideapp.html#licapp).
+ 
+## Manage Applications
+
+Go to the Applications page to get detailed information, see findings, scores, manage licenses, settings and more.
+ 
+## Set Up Environments
+
+In the Servers page, you can set a Development, QA or Production environment for each server. Select your application in the grid to compare the differences across environments as code travel and track vulnerabilities in the Overview tab. Contrast set up a shell for you to designate servers; once that’s in place, Contrast can get busy finding weaknesses.
+ 
+## Discover Vulnerabilities
+
+Go to the Vulnerabilities tab in the application's page to get a list of all vulnerabilities discovered. Then track, share and get remediation guidance for each one.
+
+### How vulnerabilities work
+
+If a vulnerability is reported and Contrast has never seen it before, Contrast creates a new vulnerability. However, if that vulnerability already exists, Contrast updates the existing entry, issue count and last seen date.
+
+<a href="assets/images/Application-vulnerabilities-tab.png" rel="lightbox" title="Vulnerabilities in the application page"><img class="thumbnail" src="assets/images/Application-vulnerabilities-tab.png"/></a>
+
+> Example: This vulnerability was reported up to Contrast five times for one server. Instead of showing up as five vulnerabilities, Contrast updates this entry and increments the count. As Contrast continues to see the same findings, the count goes up. If you dive into the Notes tab within this vulnerability, you notice a list of the servers in which this vulnerability was found.
+ 
+## Improve Your Application Score
+
+Get your application secure by remediating vulnerabilities or enabling Protection rules. We provide you with a grade to show you how well your application is performing. Visit the [Contrast Scoring Guide](https://docs.contrastsecurity.com/user_tsguideapp.html#scoreguide) for more information. 
+ 
+## Track Libraries
+
+Be aware of libraries that may be vulnerable and bring them up to date by going to the Libraries tab in the Application page. 
+ 
+Contrast provides you with a grade for the library, known Common Vulnerabilities and Exposutres (CVEs), latest version and release date, used and total classes in the library, and the application that's using the library. Contrast calculates this grade based on three things: the age of the library, how many versions behind the library is, and the number of known CVEs that affect the library.
+
+For more information, read the article on [Library Analysis](https://docs.contrastsecurity.com/user_tsguidelib.html#analysis).
+ 
+## Manage Vulnerabilities
+
+Go the Vulnerabilities page to get details on each one and get rid of weaknesses so your application isn't compromised.
```

Comment:
Can we finesse this wording more? "Go to the Vulnerabilities page to view details of each and remove weaknesses..."